scala/private/rules/scala_library.bzl

```diff
@@ -127,7 +127,6 @@ def _lib(
         jars_to_labels = jars.jars2labels,
         providers = [outputs.merged_provider, jars.jars2labels] + outputs.coverage.providers,
         runfiles = runfiles,
-        scala = scalaattr,
```

Comment:
Note above where we construct scalaattr: can we remove that? Indeed can we grep for all calls to create_scala_provider and make sure we aren’t calling that function or that it can be refactored to return a JavaInfo?